src/config-import.js

```diff
@@ -472,16 +486,23 @@ function translateMessageObject(warnings) {
   }
 }
 
+/**
+ * @param {ReadonlyArray<Entry>} entries
+ */
+function getIconEntries(entries) {
+  return entries
+    .filter((entry) => entry.filename.match(/^icons\/([^/]+)$/))
+    .sort((icon, nextIcon) => icon.filename.localeCompare(nextIcon.filename))
```

Comment:
Nit: I don't think we need to sort these.

```suggestion
  return entries.filter((entry) => entry.filename.match(/^icons\/([^/]+)$/))
```

(This diff removes the `.sort` and then re-formats it with Prettier. The `.filter` call is unchanged.)

Reply:
mhm, I think we do need to be sorted, because we're yielding. 
And that happens when the icon name has changed. If they are unsorted, then we risk getting a new variant for an icon that has already been yielded, I _think_